src/architect/account/account.utils.ts

```diff
@@ -49,7 +63,7 @@ export default class AccountUtils {
     const token_json = await app.auth.getPersistedTokenJSON();
     if (!token_json || token_json.email === 'unknown') {
       console.log(chalk.yellow('In order to access remote accounts you can login by running `architect login`'));
-      let account: Account;
+      await this.shouldListLocalAccounts(true);
```

Comment:
What does the output look like in this case? Won't it be kind of weird to show a warning log and then hard error?

Reply:
removed hard-error, and replaced with warning and then exit if neither local or remote accounts are available.